pkg/server/commands/read_assertions.go

```diff
@@ -25,9 +23,6 @@ func NewReadAssertionsQuery(backend storage.AssertionsBackend, logger logger.Log
 func (q *ReadAssertionsQuery) Execute(ctx context.Context, store, authorizationModelID string) (*openfgav1.ReadAssertionsResponse, error) {
 	assertions, err := q.backend.ReadAssertions(ctx, store, authorizationModelID)
 	if err != nil {
```

Comment:
this check is not needed as the check for assertion model found is in the [server layer](https://github.com/openfga/openfga/blob/1a846a4710fea8d87bb10a1dbee17b7194d3f23b/pkg/server/server.go#L763), not storage layer.